assets/stylesheets/bootstrap/_dropdowns.scss

```diff
@@ -183,7 +184,7 @@
     content: "";
     border-top: 0;
     border-bottom: $caret-width-base dashed;
-    border-bottom: $caret-width-base solid \9;
+    border-bottom: $caret-width-base solid \9; // IE8
```

Comment:
Again here, same , no need to mention IE8 please!  Unless you're one of the original developers of IE8 and you protest then I will consider adding the mention however if you are not a microsoft employee and have nothing to do with IE8 then I suggest we both forget about IE8.